visualdl/logic/sdk_test.cc

```diff
@@ -180,14 +143,14 @@ TEST(Audio, test) {
   for (int step = 0; step < num_steps; step++) {
     audio.StartSampling();
     for (int i = 0; i < 7; i++) {
-      vector<int64_t> shape({5, 5, 3});
-      vector<float> data;
-      for (int j = 0; j < 3 * 5 * 5; j++) {
-        data.push_back(float(rand()) / RAND_MAX);
+      vector<int32_t> shape({16000, 2, 2});
```

Comment:
What is the reason of changing these numbers? What are their meanings? Just want to learn 😄

Reply:
last time I just copied image test and change to audio. Image test they use shape [5,5,3] to represent 5x5 h and w and 3 channels (RGB). In audio, the params are [sample_rate, sample_width, number of channels]